viam-cartographer/src/slam_service/slam_service.h

```diff
@@ -209,8 +219,10 @@ class SLAMServiceImpl final : public SLAMService::Service {
     double rotation_weight = 1.0;
 
    private:
-    // kPixelSize defines the pixel size for drawing the jpeg map
-    const double kPixelSize = 0.01;
+    // resolution defines the area in meters that each pixel represent. This 
+    // is used by drawing the jpeg map as well as the resolution of the outputted 
+    // PCD
+    const double resolution = 0.05;
```

Comment:
[q] do we want to move the location of this variable?

Reply:
I'd say we should move it to the top with the other semi-configurable parameters